CMakeLists.txt

```diff
@@ -95,6 +95,10 @@ option(OGS_BUILD_GUI "Should the Data Explorer be built?" OFF)
 if(OGS_BUILD_GUI)
     add_definitions(-DOGS_BUILD_GUI)
 endif()
+option(OGS_USE_NETCDF "Add NetCDF support." ON)
+if(OGS_USE_NETCDF)
+    add_definitions(-DOGS_USE_NETCDF)
```

Comment:
Could also be moved to the Data Explorer CMakeLists.txt to narrow scope as we discussed today.

Reply:
Also moved the `option(OGS_USE_NETCDF ...)` into `OGS_BUILD_GUI` condition and tested setting the variable OFF and then enabling the GUI; it works as anticipated in the discussion.